### DISCUSSION TITLE
Transparent panel

### DISCUSSION THREAD

**Discussion by gen-failure** [regolith-linux/regolith-desktop#714] [Q&A]
I would like to call the `i3bar` with `-t` parameter to enable RGBA colors, but I can't find the right place where to set it up

**Comment by Einlar** [comment]
I managed to do this by overriding the i3-bar config, following the steps from ["How to customize a default partial"](https://regolith-desktop.com/docs/howtos/customize-i3-configuration/#how-to-customize-a-default-partial). The relevant partial is `regolith-i3-i3rocks` ([repo](https://github.com/regolith-linux/regolith-i3-config?tab=readme-ov-file#partials)) as it contains the `bar` settings that we can use to enable [transparency](https://i3wm.org/docs/userguide.html#_transparency).

1. Create the local i3 config folder if it doesn't exist: `mkdir -p ~/.config/regolith3/i3/config.d`.
2. Copy the default partial to be overridden:
```
cp /usr/share/regolith/i3/config.d/70_bar ~/.config/regolith3/i3/config.d/
```
3. Edit the `~/.config/regolith3/i3/config.d/70_bar` and add this line at the start of the `bar` block:
```
bar {
   i3bar_command i3bar --transparency 
   [...]
}
```
4. Remove the default config: `apt remove regolith-i3-i3xrocks`
5. Reload i3
